third_party/toolchains/bazel_0.23.2_rbe_windows/BUILD

```diff
@@ -0,0 +1,188 @@
+# Copyright 2018 The Bazel Authors. All rights reserved.
```

Comment:
new name of the directory "bazel_0.23.2_rbe_windows" seems better, but I would still like to know origin of these files (and answers to other questions I asked in the previous comments), ideally explained by a README.md in this directory

Reply:
Let's at least give some context in the PR description.

Reply:
added to PR description